modules/system/database/migrations/2017_10_23_000024_Db_System_Mail_Layouts_Inline_Css.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use October\Rain\Database\Schema\Blueprint;
+use October\Rain\Database\Updates\Migration;
+
+class DbSystemMailLayoutsInlineCss extends Migration
+{
+    public function up()
+    {
+        Schema::table('system_mail_layouts', function (Blueprint $table) {
+            $table->boolean('inline_css')->default(true);
```

Comment:
Store the option in a `text('options')->nullable()` jsonable property instead and we should be good

Reply:
@LukeTowers or @TimFoerster can you do this small change to get it approved?